### DISCUSSION TITLE
How to call the value of Citation key externally

### DISCUSSION THREAD

**Discussion by zoushucai** [retorquere/zotero-better-bibtex#2479] [Q&A] [answered]
Hello, I want to use JavaScript to access the citation key, but I don't know how to access it?

**Comment by retorquere** [comment]
Where do you want to use javascript?

**Reply by zoushucai** [reply]
> Where do you want to use javascript?

 tools ->  Developer -> run JavaScript  in zotreo

**Reply by retorquere** [reply]
And you want to get the citekey given an item?

**Reply by zoushucai** [reply]
> And you want to get the citekey given an item?

yes, I want to obtain the key for the selected item.

**Reply by retorquere** [reply, marked as answer]
`item.getField('citationKey')`
